### PR TITLE
[docs] Add Gradient.sweep and TileMode.decal images

### PR DESCRIPTION
**Pull request by werainkhatri**
Added missing `Gradient.sweep` images to TileMode and missing `TileMode.decal` images to Gradient.

fixes flutter/flutter#84837

## Pre-launch Checklist

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I read and followed the [Flutter Style Guide] and the [C++, Objective-C, Java style guides].
- [x] I listed at least one issue that this PR fixes in the description above.
- [ ] I added new tests to check the change I am making or feature I am adding, or Hixie said the PR is test-exempt. See [testing the engine] for instructions on writing and running engine tests.
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] I signed the [CLA].
- [x] All existing and new tests are passing.
- [ ] The reviewer has submitted any presubmit flakes in this PR using the [engine presubmit flakes form] before re-triggering the failure.

[Contributor Guide]: https://github.com/flutter/flutter/wiki/Tree-hygiene#overview
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[C++, Objective-C, Java style guides]: https://github.com/flutter/engine/blob/master/CONTRIBUTING.md#style
[testing the engine]: https://github.com/flutter/flutter/wiki/Testing-the-engine
[CLA]: https://cla.developers.google.com/
[flutter/tests]: https://github.com/flutter/tests
[breaking change policy]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes
[Discord]: https://github.com/flutter/flutter/wiki/Chat
[engine presubmit flakes form]: https://forms.gle/Wc1VyFRYJjQTH6w5A

### CODE REVIEW

**Review by chinmaygarde — APPROVED** [review]

**Comment by Hixie** [comment]
test-exempt: only affects comments

Looks like you missed "decal" though!

**Comment by werainkhatri** [comment]
> Looks like you missed "decal" though!

Sorry about that. Updated.

Should I mention in the doc that `decal` has no effect on `SweepGradient`?

File: lib/ui/painting.dart [hunk 1]
```diff
@@ -3484,6 +3486,7 @@ enum TileMode {
   ///
   /// ![](https://flutter.github.io/assets-for-api-docs/assets/dart-ui/tile_mode_mirror_linear.png)
   /// ![](https://flutter.github.io/assets-for-api-docs/assets/dart-ui/tile_mode_mirror_radial.png)
+  /// ![](https://flutter.github.io/assets-for-api-docs/assets/dart-ui/tile_mode_mirror_sweep.png)
   mirror,
 
   /// Samples beyond the edge are treated as transparent black.
```
Comment (Hixie) [inline lib/ui/painting.dart:3492]: sweep is missing from TileMode.decal below (line 3502)

File: lib/ui/painting.dart [hunk 2]
```diff
@@ -3602,6 +3606,7 @@ class Gradient extends Shader {
   /// argument. For details, see the [TileMode] enum.
   ///
   /// ![](https://flutter.github.io/assets-for-api-docs/assets/dart-ui/tile_mode_clamp_radial.png)
+  /// ![](https://flutter.github.io/assets-for-api-docs/assets/dart-ui/tile_mode_decal_radial.png)
   /// ![](https://flutter.github.io/assets-for-api-docs/assets/dart-ui/tile_mode_mirror_radial.png)
   /// ![](https://flutter.github.io/assets-for-api-docs/assets/dart-ui/tile_mode_repeated_radial.png)
   ///
```
Comment (Hixie) [inline lib/ui/painting.dart:3612]: TileMode.decal is missing from Gradient.sweep below (line 3674)

**Comment by werainkhatri** [comment]
> Why would it have no effect?

I'm sorry, It has.

**Review by Hixie — APPROVED** [review]
LGTM

**Pull request merged**